searchlib/src/vespa/searchlib/attribute/imported_search_context.h

```diff
@@ -40,7 +40,7 @@ class ImportedSearchContext : public ISearchContext {
     SearchContextParams                             _params;
 
     uint32_t getTargetLid(uint32_t lid) const {
-        return _targetLids[lid];
+        return lid < _targetLids.size() ? _targetLids[lid] : 0u;
```

Comment:
Consider making a unit test for this. E.g in imported_search_context_test.cpp. Would also be nice with a comment on why this is needed.